data_steward/validation/participants/identity_match.py

```diff
@@ -485,82 +511,318 @@ def _compare_birth_dates(person_id_set, rdr_birthdates, ehr_birthdates, match_va
 
             match_str = consts.MATCH if rdr_string == ehr_string else consts.MISMATCH
-            _record_match_value(match_values, person_id, consts.BIRTHDATE, match_str)
+            match_values[person_id] = match_str
         else:
-            _record_match_value(match_values, person_id, consts.BIRTHDATE, consts.MISMATCH)
+            match_values[person_id] = consts.MISMATCH
 
     return match_values
 
 
-def _validate_hpo_pii(hpo, rdr_values, match_values):
-    """
-    For an hpo, validate the information submitted from their PII tables.
-
-    Orders the comparisons.
+def _get_date_string(dataset):
+    date_string = dataset[-8:]
 
-    :param hpo: hpo site string
-    :param rdr_values: dictionary of values pulled from OMOP tables
-    :param match_values: current result set dictionary to update with new
-        match information
-    :return: updated match_values dictionary
-    """
-    person_ids = set()
-
-    first_names = rdr_values.get(consts.OBS_PII_NAME_FIRST)
-    middle_names = rdr_values.get(consts.OBS_PII_NAME_MIDDLE)
-    last_names = rdr_values.get(consts.OBS_PII_NAME_LAST)
-    person_id_set, match_values = _compare_name_fields(hpo, first_names, middle_names, last_names, match_values)
-    person_ids.update(person_id_set)
-
-    email_addresses = rdr_values.get(consts.OBS_PII_EMAIL_ADDRESS)
-    person_id_set, match_values = _compare_email_addresses(hpo, email_addresses, match_values)
-    person_ids.update(person_id_set)
-
-    phone_numbers = rdr_values.get(consts.OBS_PII_PHONE)
-    person_id_set, match_values = _compare_phone_numbers(hpo, phone_numbers, match_values)
-    person_ids.update(person_id_set)
-
-    streets_one = rdr_values.get(consts.OBS_PII_STREET_ADDRESS_ONE)
-    streets_two = rdr_values.get(consts.OBS_PII_STREET_ADDRESS_TWO)
-    cities = rdr_values.get(consts.OBS_PII_STREET_ADDRESS_CITY)
-    states = rdr_values.get(consts.OBS_PII_STREET_ADDRESS_STATE)
-    zips = rdr_values.get(consts.OBS_PII_STREET_ADDRESS_ZIP)
-    person_id_set, match_values = _compare_address_fields(hpo, streets_one, streets_two, cities, states, zips,
-                                                          match_values)
-    person_ids.update(person_id_set)
-
-    rdr_birthdate = rdr_values.get(consts.OBS_PII_BIRTH_DATETIME)
-    ehr_birthdate = rdr_values.get(consts.OBS_EHR_BIRTH_DATETIME)
-    match_values = _compare_birth_dates(person_ids, rdr_birthdate, ehr_birthdate, match_values)
+    if re.match(consts.DRC_DATE_REGEX, date_string):
+        return date_string
 
-    return match_values
+    return datetime.now().strftime(consts.DRC_DATE_FORMAT)
 
 
-def match_participants(dataset):
+def match_participants(project, rdr_dataset, ehr_dataset, dest_dataset_id):
     """
     Entry point for performing participant matching of PPI, EHR, and PII data.
 
-    :param dataset:  The string name of the dataset to use when matching participants
+    :param project: a string representing the project name
```

Comment:
It will be informative if the parameters rdr_datasetr, ehr_dataset, dest_dataset_id  were added to the doc string.

Reply:
Good catch.  Done.